### ISSUE TITLE
Bug Introduced in version 19.74 for users with "Categorize modules" set to "No"

### ISSUE THREAD

**Issue by rverrips** [virtualmin/authentic-theme#1552]
Hi 

I have some users who have access to only a few modules within Webmin - Because they only have a few modules to simplify things I set "Categorize modules" to "No" in the "User Interface options" in "Webmin Users" (See screenshot)

![Screen Shot 2021-06-11 at 4 55 06 PM](https://user-images.githubusercontent.com/749015/121752607-1dff1300-cad6-11eb-9d32-183ad5477582.png)

This worked fine up until version 19.73 of the theme and the users would get a single listing in their options in the navigation window.  

From version 19.74 the single listing still shows, but no longer loads the module when clicked and rather just toggles what looks like a menu indicator.  

The user still only has access to modules set, and can "find" them using search, so just seems to be a navigation bar issue, not permissions, etc. 

The attached screenflow shows how it worked in version 19.73 and stopped working after upgrading to version 19.74 (I've tried all versions up to 19.80 since and still this issue persists) 

Thanks

Roy

![webmin-themes-issue-2](https://user-images.githubusercontent.com/749015/121754054-42102380-cad9-11eb-9c96-30e66392ef4c.gif)

**Comment by iliarostovtsev** [comment]
Hello,

Thanks for the heads up.

What a beautiful bug report, I must say! I will have a look at this issue tomorrow.

**Comment by iliarostovtsev** [comment]
Give this patch a try please https://github.com/virtualmin/authentic-theme/commit/e379d0811119e7eaca81aa4ba4d8f8be99f1e375

**Comment by rverrips** [comment]
Thank you SO much for the quick response, @iliarostovtsev 

The patch e379d08 fixed the issue in version 19.74 
I also tested with the most current version stable version 19.80 and installed the patch and once I installed the patch it worked there too
I then had the theme update itself to the development which installed 19.80-4 and it worked there "out of the box" without needing to implement the fix. 

I am assuming the fix will be included in the next stable release so I'm closing this issue as I'm very satisfied with the resolution.

**Thank you again** for the quick response and **GREAT** theme that makes Webmin an even better product/experience.
